app/routes/convert.py

```
@@ -380,13 +397,19 @@ def __init__(self, message: str):
         "Archive is not a single handheld-ROM archive produced by this tool "
         "(corrupt, multi-file, or holds no ROM)",
     ),
+    SkipReason.NKIT_BAD_EXTENSION: (
+        400,
+        "nkit_restore requires an NKit-shrunk GameCube/Wii image "
+        "(.nkit.iso, .nkit.gcz)",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)